### PR TITLE
Sigma Prime AUDIT Response

### PR DESCRIPTION
**Pull request by Sidu28**

### CODE REVIEW

**Comment by Sidu28** [comment]
Here is the audit response doc: https://docs.google.com/document/d/1sRT3HpEsfK-Uixakk0LFhJG58rdSJrCZKZNm2T5kNuQ/edit#

File: src/contracts/pods/DelayedWithdrawalRouter.sol [hunk 1]
```diff
@@ -56,7 +56,8 @@ contract DelayedWithdrawalRouter is Initializable, OwnableUpgradeable, Reentranc
      * @notice Creates a delayed withdrawal for `msg.value` to the `recipient`.
      * @dev Only callable by the `podOwner`'s EigenPod contract.
      */
-    function createDelayedWithdrawal(address podOwner, address recipient) external payable onlyEigenPod(podOwner) {
+    function createDelayedWithdrawal(address podOwner, address recipient) external payable onlyEigenPod(podOwner) onlyWhenNotPaused(PAUSED_DELAYED_WITHDRAWAL_CLAIMS) {
+        require(recipient != address(0), "DelayedWithdrawalRouter.createDelayedWithdrawal: recipient cannot be zero address");
```
Comment (ChaoticWalrus) [inline src/contracts/pods/DelayedWithdrawalRouter.sol:60]: can we add unit test coverage for these changes?

**Comment by ChaoticWalrus** [comment]
Added a few comments to the doc + the PR. Seems OK, would like to see just a little more polish

File: src/contracts/pods/EigenPod.sol [hunk 2]
```diff
@@ -214,10 +214,8 @@ contract EigenPod is IEigenPod, Initializable, ReentrancyGuardUpgradeable, Eigen
         // set the status to active
         validatorStatus[validatorIndex] = VALIDATOR_STATUS.ACTIVE;
 
-        // Sets "hasRestaked" to true if it hasn't been set yet. 
-        if (!hasRestaked) {
-            hasRestaked = true;
-        }
+        // Sets "hasRestaked" to true
+        hasRestaked = true;
```
[the remark this answers is not in the document]
Comment (Sidu28) [reply src/contracts/pods/EigenPod.sol:218]: I kinda agree too, its way more clear.  They are technically correct that its "necessary".  I'm honestly ok reverting to what it was

File: src/contracts/pods/DelayedWithdrawalRouter.sol (hunk 1 above)
Comment (Sidu28) [reply src/contracts/pods/DelayedWithdrawalRouter.sol:60]: added: [097c530](https://github.com/Layr-Labs/eigenlayer-contracts/pull/30/commits/097c530a90d7eac45accb84c575c951fa4b155e5)

**Review by ChaoticWalrus — APPROVED** [review]
LGTM

**Pull request merged**